### DISCUSSION TITLE
Details on Calculating and Maintaining the Optimum Staking Ratio

### DISCUSSION THREAD

**Discussion by boredray** [teaxyz/white-paper#121] [Q&A] [answered]
In the white paper, it's mentioned that the tea Protocol aims to implement an optimum staking ratio to prevent a small number of highly-staked packages from collecting most staking rewards. Could you provide more details on how this optimum staking ratio will be calculated and adjusted? Additionally, what measures will be in place to ensure a fair distribution of staking rewards among packages with varying degrees of staked TEA tokens?

**Comment by thomas-borrel** [comment, marked as answer]
Hi @boredray - Great questions!
The optimum staking ratio is a network parameter that can be edited through governance.
The way the optimum staking ratio functions is that no project can receive more than 1% of the epoch rewards emissions.
The staking ratio is calculated on a per-project basis based on the number of tokens staked towards the project. If that number exceeds 1%, it gets capped - i.e. as more participants stake this package, their yield decreases since the amount of staking rewards distributed to the project will stay at 1% of the per-epoch emissions, while the number of accounts the amount needs to be distributed to increases. The use of an algorithmic OSR (as an alternative to a set percentage) would have empowered large token holders to materially influence the rewards distribution across all projects, instead of the project(s) they stake on.
Staking rewards distribution is dictated by the number of tokens staked to each individual projects and each participant's contribution to the pool, which ensures a fair distribution of rewards across all projects.
